thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashService.js

```diff
@@ -57,7 +65,7 @@ HashService.prototype = {
     console.log('hash service.update');
     console.log(paramsToUpdate)
     for (var key in paramsToUpdate) {
-      this.params[key] = paramsToUpdate[key];
```

Comment:
`Object#hasOwnProperty()` should be used here if you're sticking with `for..in`, or `Object#keys` on paramsToUpdate if you want to iterate with  `Array#forEach`